rllib/algorithms/algorithm.py

```diff
@@ -826,60 +826,43 @@ def setup(self, config: AlgorithmConfig) -> None:
                 spaces=self.env_runner_group.get_spaces(),
                 inference_only=False,
             )
-            agg_cls = ray.remote(
-                num_cpus=1,
-                # TODO (sven): Activate this when Ray has figured out GPU pre-loading.
-                # num_gpus=0.01 if self.config.num_gpus_per_learner > 0 else 0,
-                max_restarts=-1,
-            )(AggregatorActor)
+
+            pg = self.learner_group._placement_group
+            agg_cls_list = []
+            aggregator_index = 0
+            self._aggregator_actor_to_learner = {}
+            for i in range(self.config.num_aggregator_actors_per_learner):
```

Comment:
nit: rename `i` to `agg_idx` and `j` to `learner_idx` for clarity. These one-char vars are always a big source of future bugs :)

Reply:
Ah, sorry, get rid of `i` altogether. It's not needed. Already covered by the other var: `aggregator_index`.